### PR TITLE
Goals form

### PR DESCRIPTION
**Pull request by ymik0410**

### CODE REVIEW

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -1,6 +1,16 @@
 <%= simple_form_for(@user) do |f| %>
   <%= f.input :introduction, label: 'Introduction', placeholder: 'Some words about yourself' %>
   <%= f.input :goal_description, label: 'Describe your goal'  %>
```
Comment (casantadu) [inline app/views/users/_form.html.erb]: Could you please put: placeholder: 'I want to achieve this goal because...'

**Review by casantadu — CHANGES_REQUESTED** [review]
Could you please add placeholder for goal description

File: app/views/users/_form.html.erb (hunk 1 above)
Comment (ymik0410) [reply app/views/users/_form.html.erb]: I have not seen your comment but wrote exactly the same phrase!

Comment (ymik0410) [reply app/views/users/_form.html.erb]: done

**Pull request merged**